packages/components/src/components/button-toggle/button-toggle.jsx

```diff
@@ -5,7 +5,17 @@ import HighlightWrapper from './button-highlight-wrapper.jsx';
 import Button from '../button/button.jsx';
 import Counter from '../counter/counter.jsx';
 
-const ButtonToggle = ({ buttons_arr, className, has_rounded_button, id, is_animated, name, onChange, value }) => {
+const ButtonToggle = ({
+    buttons_arr,
+    button_large,
```

Comment:
Just `large` is okay. Since `packages/components/src/components/button/button.jsx` file follows that, let's follow that here as well.